### ISSUE TITLE
Update FRAC information on PHI-base

### ISSUE THREAD

**Issue by CuzickA** [PHI-base/data#16]
Currently on PHI-base under the anti-infectives tab we have a link to FRAC for the 2018 document. Please could this be updated to the 2022 document. 

https://www.frac.info/docs/default-source/publications/frac-code-list/frac-code-list-2022--final.pdf?sfvrsn=b6024e9a_2

We will also want this information to be displayed on PHI-base 5.

Tagging @martin2urban and @jseager7 as I'm not sure who will make this update, thanks.

(It is also required for a new AE to record a FRAC code that I am planning to request soon).

**Comment by jseager7** [comment]
I think this is for Martin since I don't maintain the PHI-base 4 website.

**Comment by CuzickA** [comment]
We decided that it was not necessary to update the FRAC file on the PHI-base 4 website as this may be a difficult task. The new FRAC code AE will not be displayed on PHI-base 4.

In PHI-base 5 the anti-infective tab already points at the FRAC webpage which will contain the most up to date version of the FRAC code list.
![image](https://github.com/PHI-base/data/assets/11753634/e3e6a3b9-9bb8-4000-a6bf-d29cfab5d2f0)
![image](https://github.com/PHI-base/data/assets/11753634/eb49173b-5284-465e-88be-1db3814ac22e)
https://www.frac.info/

Closing ticket.
